src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,8 +1,46 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Arrays;
+
 public class SalaryInfo {
 
+    private static final DateTimeFormatter DATE_FORMATTER =
+            DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder report = new StringBuilder("Report for period ").append(dateFrom)
+                                                                      .append(" - ")
+                                                                      .append(dateTo)
+                                                                      .append("\n");
+        ArrayList<String> processedNames = new ArrayList<>();
```

Comment:
```suggestion
        List<String> processedNames = new ArrayList<>();
```
use more abstract link when possible